xml/System/Guid.xml

```diff
@@ -1823,9 +1941,9 @@
         <Parameter Name="destination" Type="System.Span&lt;System.Byte&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">When this method returns, the GUID as a span of bytes.</param>
+        <summary>Tries to write the current GUID instance into a span of bytes.</summary>
+        <returns><see langword="true" /> if the GUID is sucessfully written to given span; <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the GUID is sucessfully written to the specified span; <see langword="false" /> otherwise.</returns>
```